src/screens/Auth/AuthScreen.js

```diff
@@ -28,6 +29,60 @@ class AuthScreen extends React.Component {
 		headerTruncatedBackTitle: _('back'),
 	});
 
+	// https://facebook.github.io/react-native/docs/alert.html#ios
+	static getAlertButtons() {
+		if (Platform.OS === 'ios') {
+			return [
+				{
+					text: _('ok'),
+				},
+			];
+		}
+
+		return [
+			{},
+			{
+				text: _('ok'),
+			},
+		];
+	}
+
+	static openWrongExt() {
+		Alert.alert(
+			t('bad_login_password'),
+			t('wrong_ext'),
+			[
+				{
+					text: t('i_am_cas'),
+					style: Platform.OS === 'ios' ? 'destructive' : 'neutral',
+					onPress: () => AuthScreen.openWrongCas(),
+				},
+				...AuthScreen.getAlertButtons(),
+			],
+			{
+				cancelable: true,
```

Comment:
iOS doesn't provide a "Cancel" option by default. You need to add a button with "cancel" style. (Usually placed in first according to RN documentation)

Reply:
The option cancelable allows only the user to cancel the alert by clicking outside ^^